script/Arcueid/Day 7/07_00_ARC07_3_1_4.txt

```diff
@@ -6,12 +6,12 @@ This is a good opportunity. No, this is the best chance I'm going to get.
 [sha:b1854467dc79b7e7214eb4203b06d241d8bbe685]{
 -- Page 10019, Offset 18143.
 -- 実はものすごく気になっていたんだけど、
-To be honest, there was something that I was really curious about...
+To be honest, there was something I was really curious about...
 }
 [sha:972d5531813dde407afd180acd6b6d259ccb08b2]{
 -- Page 621, Offset 18144. Mods: @n.
 -- 「ヴローヴと戦っていた時に着てた服、
-"Those clothes that you wore when fighting Vlov...
+"Those clothes you wore when fighting Vlov...
 }
 [sha:7f68ce8810566ac434b18de7acff93ae1330dc3d]{
 -- Page 621, Offset 18145. Glued.
```

Comment:
github is being irritating, but this should be "show _them_ to me". I'm guessing at one point the line was "that dress you wore"